src/components/CreateCourse/CreateCourse.tsx

```diff
@@ -30,10 +34,43 @@ const authorNameErrorLabelMessage = 'length should be at least 2 characters';
 const courseAuthorListErrorLabelMessage = 'Add authors to this list';
 const AUTHOR_LIST_IS_EMPTY_MESSAGE = 'Author list is Empty';
 
-export const CreateCourse: React.FC = () => {
-	checkLogIn();
+const COURSE_FORM_TITLE = 'Main Info';
+
+const TITLE_NAME_VALUE = 'title';
+const TITLE_LABEL_TEXT = 'Title*';
+const INPUT_TITLE_PLACEHOLDER_TEXT = 'Input title';
+
+const DESCRIPTION_NAME_VALUE = 'description';
+const DESCRIPTION_LABEL_TEXT = 'Description*';
+const INPUT_DESCRIPTION_PLACEHOLDER_TEXT = 'Input description';
+
+const DURATION_PART_TITLE = 'Duration';
+const DURATION_NAME_VALUE = 'duration';
+const DURATION_LABEL_TEXT = 'Duration*';
+const INPUT_DURATION_PLACEHOLDER_TEXT = 'Input duration';
+
+const AUTHORS_PART_TITLE = 'Authors';
+const AUTHOR_NAME_VALUE = 'authorName';
+const AUTHORS_LABEL_TEXT = 'Author Name';
+const INPUT_AUTHORS_PLACEHOLDER_TEXT = 'Input author name';
 
+const CREATE_AUTHOR_BUTTON_TEXT = 'CREATE AUTHOR';
+
+const COURSE_AUTHOR_LIST_TITLE = 'Course Authors*';
+
+const CANCEL_BUTTON_TEXT = 'CANCEL';
+const ADD_COURSE_BUTTON_TEXT = 'ADD COURSE';
```

Comment:
you can create a constants file for this component and move this constants there)